clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -165,7 +165,7 @@ public abstract class StoreIngestionTask implements Runnable, Closeable {
 
   protected static final long WAITING_TIME_FOR_LAST_RECORD_TO_BE_PROCESSED = MINUTES.toMillis(1); // 1 min
 
-  private static final int MAX_CONSUMER_ACTION_ATTEMPTS = 5;
+  static final int MAX_CONSUMER_ACTION_ATTEMPTS = 5;
```

Comment:
Nitpick: shall we add a 'protected' keyword?

Reply:
I think it is intentionally package private meant for unit test. Maybe add `@VisibleForTesting` to make it explicit if we already have the annotations dependency (guava) in the module.

Reply:
Yeah, in a previous review I was told to make it package-private which is even more strict than `protected`, and not use `@VisibleForTesting` ([link](https://github.com/linkedin/venice/pull/1052#discussion_r1669379315)).